lib/peep.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'database_connection'
+
+class Peep
+  def self.all
+    result = DatabaseConnection.query("SELECT * FROM peeps ORDER BY peep_id DESC;")
+    result.map do |peep| 
+      Peep.new(
+        peep_id: peep['peep_id'], 
+        peep_text: peep['peep_text'], 
+        peeped_on: peep['peeped_on'],
+        user_id: peep['user_id']
+      )
+    end
+  end
+
+  def self.create(peep_text:, user_id:)
```

Comment:
Good implementation of the user_id to join the user table, clear and concise code